src/index.js

```diff
@@ -42,7 +40,6 @@ export default function supernova(env) {
       const rect = useRect();
 
       const [pageInfo, setPageInfo] = useState(() => ({ top: 0, height: 100 }));
-      const [muiParameters] = useState(() => muiSetup());
```

Comment:
We had it here so we don't have to run muiSetup everytime the render `uesEffect` is triggered. Is it not working when it was here or can we keep it here as a state?

Reply:
ok sure